features/add_a_job.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Add a job
+  As a claimant,
+  I want to report jobs I am interested in or applying for
+  so that I can continue to receive UC and avoid sanctions
+
+  Scenario: Add an application
+    When I add a job application
+    Then it should show on my dashboard
+
+  Scenario: Validation
+    When I add a job application without a title
+    Then I should see a validation error
+
+  Scenario: Multiple claimants add job application
+    When "John" adds a job application
+    And "Anna" adds a job application
+    Then "John" should only see his job application
+    And "Anna" should only see her job application
```

Comment:
This reads a lot like a test script, as opposed to a description of the desired behaviour.

What about something more like:

```
When multiple claimants have added job applications to their accounts
Then each claimant should only be able to see their respective job applications
```